techniques/failures/F10.html

```diff
@@ -52,85 +49,68 @@
 </section>
 	
 
-<section id="description"><h2>Description</h2>
-      <p>When content includes multiple formats, one or more user agents or plug-ins
-                        are often needed in order to successfully present the content to users. For
-                        example, a page that includes HTML, SVG, SMIL and XForms may require a
-                        browser to load as many as three different plug-ins in order for a user to
-                        successfully interact with the content. Some plug-ins create a common
-                        situation in which the keyboard focus can become "stuck" in a
-                        plug-in, leaving a keyboard-only user with no way to return to the
-                        other content. </p>
-   </section><section id="examples"><h2>Examples</h2>
+<section id="description"><h2>解説</h2>
+      <p>コンテンツに複数のフォーマットが含まれている場合、そのコンテンツを利用者に正しく提示するのに、しばしば一つ以上のユーザエージェント又はプラグインが必要となる。例えば、HTML、SVG、SMIL、XForms を含んだウェブページは、利用者がコンテンツと情報のやり取りを行えるように、ブラウザに三つもの数のプラグインを要求することがある。一部のプラグインでは、キーボードのフォーカスがプラグインで「閉じ込められ」て、キーボードのみの利用者は他のコンテンツに戻ることができないという共通の状況が発生する。</p>
+   </section><section id="examples"><h2>事例</h2>
       <ul>
          <li> 
-               <strong>A plug-in traps the user: </strong>A user tabs into a
-                                plug-in and is unable to return to content outside the plug-in
-                                content with the keyboard. The user has to restart their browser in
-                                order to regain control and navigate to a new page and is unable to
-                                access any content that appears beyond the plug-in content. </li>
+               <strong>利用者を閉じ込めるプラグイン: </strong>利用者が Tab キーを押してプラグインに入ると、プラグインの外のコンテンツにキーボードを使って戻ることができなくなる。元の状態に戻って、新しいウェブページにナビゲートするには、利用者はブラウザを再起動しなければならず、このプラグインのコンテンツの後に続くコンテンツへは、アクセスできない。</li>
       </ul>
    </section>
 
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../general/G21">G21: Ensuring that users are not trapped in content</a></li>
       <li><a href="../client-side-script/SCR20">SCR20: Using both keyboard and other device-specific functions</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <ol>
-            <li> Using a keyboard, navigate through the content. </li>
-            <li> Check to see that the keyboard focus is not "trapped" and it is
-                                    possible to move keyboard focus out of the plug-in content
-                                    without closing the user agent or restarting the system. </li>
+            <li>キーボードを使って、コンテンツ内を行き来する。</li>
+            <li>キーボードフォーカスが「閉じ込め」られることはなく、かつ、ユーザエージェントを閉じたり、OS を再起動したりすることなく、キーボードフォーカスをプラグインのコンテンツの外に移動させることが可能であることを確認する。</li>
```

Comment:
check の訳を他のテクニックと合わせました。
```suggestion
            <li>キーボードフォーカスが「閉じ込め」られることはなく、かつ、ユーザエージェントを閉じたり、OS を再起動したりすることなく、キーボードフォーカスをプラグインのコンテンツの外に移動させることが可能であることをチェックする。</li>
```